client/src/inside/components/page/ehr-helper.js

```diff
@@ -468,6 +469,20 @@ export default class EhrHelpV2 {
         let msg = label + ' is required'
         dialog.errorList.push(msg)
       }
+      if (eDef.validation && validations[eDef.validation]) {
+        if(!validations[eDef.validation](inputs[eKey])) {
+          const errKey = `${eDef.validation.toUpperCase()}_ERROR`
+          if (dbDialog) console.log('errKey >> ', errKey, 'eKey >> ', eKey)
+          if(eKey.includes('prn')) {
+            const prnString = 'PRN values'
+            if(!dialog.errorList.includes(validations[errKey](prnString))) {
+              dialog.errorList.push(validations[errKey](prnString))
+            } 
+          } else {
+            dialog.errorList.push(validations[errKey](eKey))
```

Comment:
Please remove this prn stuff. I will create a ticket for dialog having too many errors.